lib/as2/client.rb

```diff
@@ -172,7 +172,7 @@ def format_body_v0(document_content, content_type:, file_name:)
       document_payload << "Content-Transfer-Encoding: base64\r\n"
       document_payload << "Content-Disposition: attachment; filename=#{file_name}\r\n"
       document_payload << "\r\n"
-      document_payload << Base64.strict_encode64(document_content)
+      document_payload << base64_encode(document_content)
```

Comment:
we are now canonicalizing line-endings after encoding.

this will not be a behavior change for existing configs (which all use RFC-4648, aka `.strict_encode`) since RFC-4648 base64 will not contain any line endings.